packages/client/modules/teamDashboard/components/AgendaList/AgendaList.tsx

```diff
@@ -69,7 +69,7 @@ const AgendaList = (props: Props) => {
     } else {
       const offset = source.index > destination.index ? -1 : 1
       sortOrder =
-        (agendaItems[destination.index + offset].sortOrder + destinationItem.sortOrder) / 2 +
+        (agendaItems[destination.index + offset]!.sortOrder + destinationItem.sortOrder) / 2 +
```

Comment:
please don't assert non-null-ness here, destination and agendaItems come from 2 different sources